cypress.config.js

```diff
@@ -1,21 +1,35 @@
-const { defineConfig } = require("cypress");
-const faker = require('faker');
+const { defineConfig } = require('cypress');
+const { faker } = require('@faker-js/faker');
 
 module.exports = defineConfig({
   e2e: {
     baseUrl: 'https://conduit.mate.academy/',
     setupNodeEvents(on, config) {
       on('task', {
         generateUser() {
-          const email = faker.internet.email();
+          const email = faker.internet.exampleEmail();
           const randomNumber = Math.floor(Math.random(1000) * 1000);
           return {
-            username: faker.name.firstName() + randomNumber,
+            username: faker.person.firstName() + randomNumber,
             email: email.toLowerCase(),
             password: '12345Qwert!'
           };
         }
       });
+
+      on('task', {
+        generateArticle() {
+          const title = faker.lorem.words();
+          const description = faker.lorem.sentence();
+          const body = faker.lorem.sentences();
+
+          return {
+            title,
+            description,
+            body
+          };
+        }
+      });
     }
```

Comment:
The function `setupNodeEvents` is not a valid configuration option for Cypress. You seem to be trying to define custom tasks to generate user and article data. This should be done in the `plugins/index.js` file using the `on('task', ...)` syntax inside the `module.exports` function. Please move the code related to the tasks to the appropriate place.